ports/espressif/boards/waveshare_esp32s2_pico/mpconfigboard.mk

```diff
@@ -0,0 +1,17 @@
+USB_VID = 0x239a
```

Comment:
This is Adafruit's VID. This is in no way official from Waveshare. Can you link me to the support ticket?

Reply:
As you wish, Attaching .eml file & screenshot.
![image](https://user-images.githubusercontent.com/21365918/182444036-7bab4062-89dc-4dfd-ba1b-29aa2c4f5a21.png)
https://github.com/adafruit/circuitpython/files/9244660/Waveshare.Re.ESP32-S2-Pico.USB_VID.USB_PID.-.Waveshare.support%40waveshare.com.-.2022-08-02.0420.zip
In zip cuz github doesn't allow .eml
Should you wish for me to reply to the support ticket, please let me know.

Reply:
Screenshot is enough. Thanks. I won't merge this with an Adafruit VID. I'd suggest requesting one from Espressif.

Reply:
@SSYYL @waveshare do not use adafruit VID for your products, they are allocated for adafruit products only

Reply:
> I'd suggest requesting one from Espressif.

I have requested one in https://github.com/espressif/usb-pids/pull/55.

Reply:
Got 0x810a (for cp) and 0x8109 (for uf2) from espressif.
![chris](https://user-images.githubusercontent.com/21365918/182533167-617aa523-6485-477f-87b1-9a4021e4a823.png)

Reply:
Thank you @ladyada. I quite agree with you. I have told the colleague in charge of this board.